site/topics/logic/truth-tables.rst

```diff
@@ -2,14 +2,91 @@
 Truth Tables
 ************
 
+* Much of the following will likely be a review of already well understood concepts
+* This content is covered for completeness, but will be kept at a relatively high level
+
 
 
 Boolean Logic
 =============
 
+* Boolean logic is a form of algebra that operates on boolean values that can take on only two states
+* These states are typically called *True* and *False*
 
-Boolean Algebra
----------------
+    * Depending on context, these are sometimes referred to as *On*/*Off*, ``1``/``0``, or *high voltage*/*low voltage*
+
+
+* Within the algebra, the *operands* are values that take on one of these two states
+* The *operators* act on these operands to produce a new boolean value
+
+
+.. note::
+
+    If this terminology is throwing you off, remember that this is like the integer operators/operands you are familiar
+    with. Consider the expression :math:`1 + 2`. The operands here are integers :math:`1` and :math:`2` and the operator
+    is :math:`+`, which means addition. Here, addition is an operator that takes two integer values and produces a new
+    integer value.
+
+    With boolean logic, the operands can have one of two boolean values as opposed to an infinite number, like with
+    integers, and the operators will take two boolean values to produce a new boolean value.
```

Comment:
Is this unnecessary/overkill?

Reply:
This may be the first time the students really see mathematical operations with things other than numbers (or things representative of numbers), I don't think it's bad to call out the general terms with some examples. Like they may not have seen sets, or really thought about differentiation being an operator on functions.  
I'd remove the last sentence though, it's got some _technically_ false statements. There's operators that don't produce a value in the same domain as the inputs, nullary operators, n-ary operators, all kinds of crap.